usr/share/rear/restore/FDRUPSTREAM/default/260_copy_log_and_report.sh

```diff
@@ -1,14 +1,59 @@
-# upstream.log and upstream.rpt on the ReaR system contains valuable 
+# upstream.log and upstream.rpt on the ReaR system contain valuable 
 # information about the disaster recovery, so we archive *.log and *.rpt
 # to the restored system.
 
-EXTENSION="restore_$( date +%F_%T )"
+PREFIX="rear_$( date +%F_%T_%N )"
+SERVICENAME=( $(ps -ef | grep uscmd1 | grep servicename | grep -Po 'servicename=\K[^"]+') )
+REARLOGPATH="$FDRUPSTREAM_DATA_PATH/rear/logs"
+
+# If SERVICENAME is empty, then FDR/Upstream is not running and we won't 
+# be able to find the log files.
+if [[ ! ${SERVICENAME[@]} ]]; then 
+	echo
+	LogPrintError "***************"
+	LogPrintError "***************"
+	LogPrintError "No FDR/Upstream services were detected.  Unable to automatically copy logs and reports to the recovered system."
+	LogPrintError "Before rebooting, copy any necessary logs and reports from $FDRUPSTREAM_DATA_PATH into the $TARGET_FS_ROOT file tree."
+	LogPrintError "***************"
+	LogPrintError "***************"
+	echo
```

Comment:
@rowens275 
out of curiosity:
I wonder what the reason behind is why you have the plain `echo` lines?

While usr/sbin/rear is running both stdout and stderr
are redirected into ReaR's log file,
cf. "What to do with stdin, stdout, and stderr" in
https://github.com/rear/rear/wiki/Coding-Style

So plain `echo` lines result empty lines in ReaR's log file
and in debugscript mode `-D` one gets in the log
```
++ echo

```

It doesn't hurt but to me it looks a bit strange.

Reply:
I was just trying to get certain messages to stand out on the screen.  I thought that since I didn't use "LogPrint", that it would not appear in the log.  But I must not have tested that.  It's ok with me if the blank lines appear in the log, but the main purpose was for them to be an eye-catcher during interactive use of ReaR.